js/events/utils/add_namespace.js

```diff
@@ -0,0 +1,19 @@
+import errors from '../../core/errors';
+
+const addNamespace = (eventNames, namespace) => {
+    if(!namespace) {
+        throw errors.Error('E0017');
+    }
+
+    if(Array.isArray(eventNames)) {
+        return eventNames
+            .map(eventName => `${eventName}.${namespace}`)
+            .join(' ');
+    } else if(eventNames.indexOf(' ') !== -1) {
```

Comment:
```suggestion
    }
    
    if(eventNames.indexOf(' ') !== -1) {
```
No need for `else` since previois `if` block contains `return` statement